dynatrace-oneagent-operator/README.md

```diff
@@ -69,12 +80,28 @@ You can then check for the latest version by searching your Helm repositories fo
 $ helm search repo dynatrace-oneagent-operator
 ```
 
-To then update to the latest version run this command and do not forget to add the `reuse-values` flag to keep your configuration
+To update to the latest version apply the latest version of the CRD attached to [the latest release](https://github.com/Dynatrace/dynatrace-oneagent-operator/releases/latest) and run this command.
+Do not forget to add the `reuse-values` flag to keep your configuration
+
+##### Kubernetes
+```
+$ kubectl apply -f https://github.com/Dynatrace/dynatrace-oneagent-operator/releases/latest/download/dynatrace.com_dynakubes.yaml
+$ helm upgrade dynatrace-oneagent-operator dynatrace/dynatrace-oneagent-operator -n dynatrace --reuse-values
+```
+
+##### OpenShift
+```
+$ oc apply -f https://github.com/Dynatrace/dynatrace-oneagent-operator/releases/latest/download/dynatrace.com_dynakubes.yaml
+$ helm upgrade dynatrace-oneagent-operator dynatrace/dynatrace-oneagent-operator -n dynatrace --reuse-values
+```
 
+##### OpenShift 3.11
 ```
+$ oc apply -f https://github.com/Dynatrace/dynatrace-oneagent-operator/releases/latest/download/dynatrace.com_dynakubes-v1beta1.yaml
```

Comment:
super nitpick: `dynatrace.com_dynakubes-v1beta1.yaml` is rather ugly, it has a `.` a `_` and `-`  in the name, I would just choose one, 
so `dynatrace.com.dynakubes.v1beta1.yaml`, `dynatrace-com-dynakubes-v1beta1.yaml`, `dynatrace_com_dynakubes-v1beta1.yaml`

Reply:
Yup I totally understand.
Also was kinda annoyed by that but I used the schema I got from here:
https://github.com/Dynatrace/dynatrace-operator/tree/master/config/crd/default/bases
and just appended the v1beta1

What do you think about `dynatrace.com_dynakubes_v1beta1.yaml` ?

Reply:
it looks good 😄 
(but honestly, my favorite is still `dynatrace.com.dynakubes.v1beta1.yaml` 😛 , it makes my nonexistent OCD happy)

Reply:
![image](https://user-images.githubusercontent.com/31651557/130619764-d34edf71-8e08-4d48-8f51-224788c09b0c.png)
also this how the crd is named in kubernetes

Reply:
The name here is the one which gets auto-generated by operator-sdk's make file
https://github.com/Dynatrace/dynatrace-operator/tree/master/config/crd/default/bases

If you're ok with it I would like to keep the names since it makes to automation to add them to the release artefacts easier (just append something instead of exchanging dots and underlines)

Reply:
already approved it 20 minutes ago 😉